script/cri/test.sh

```diff
@@ -78,10 +79,11 @@ RUN apt install -y --no-install-recommends git make gcc build-essential jq && \
     curl https://dl.google.com/go/go1.15.6.linux-\${TARGETARCH:-amd64}.tar.gz \
     | tar -C /usr/local -xz && \
     go get -u github.com/onsi/ginkgo/ginkgo && \
-    git clone -b v1.19.0 https://github.com/kubernetes-sigs/cri-tools \
+    git clone https://github.com/kubernetes-sigs/cri-tools \
```

Comment:
FYI these have been recently cleaned up... https://github.com/containerd/containerd/blob/master/Makefile#L283-L284

Reply:
Thanks! I'll work on following-up PRs for refactoring this.